Entities/Industry/Drill/Drill.as

```diff
@@ -438,8 +444,8 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 
 	if (customData == Hitters::water)
 	{
-		s16 current_heat = this.get_u8(heat_prop) - heat_max*0.7f;
-		if (current_heat < 0) current_heat= 0;
+		s16 current_heat = this.get_u8(heat_prop) - heat_max * 0.5f;
```

Comment:
Overall it might be a good idea to refactor all those heat vars as global constants (e.g. `water_heat_reduction` or whatever). Not necessarily in this PR, though

Reply:
done